### DISCUSSION TITLE
Vendor chunk naming ignores webpackChunkName comments

### DISCUSSION THREAD

**Discussion by MilosRasic** [webpack/webpack#14000] [Q&A]
# Bug report

**What is the current behavior?**
For a dynamic import like this
```
import(/* webpackChunkName: "account" */ /* webpackPrefetch: true */ './Components/Account/Account'),
```
Webpack 5 will generate the following source code chunk for me:
account.e5d772cdaf3c5f18ee93.js

and a vendor chunk that looks something like:
323.fee73acb2259d6f54633.js

**If the current behavior is a bug, please provide the steps to reproduce.**

1. Try a dynamic import with `webpackChunkName` magic comment
2. Do not specify `optimization` config or use `SplitChunksPlugin`
3. Run the build
4. Check the resulting chunk files and try to find the vendor chunk for your source chunk

**What is the expected behavior?**
Previously, with webpack 4, I'd get files that look like:
account.e5d772cdaf3c5f18ee93.js
account~vendor.fee73acb2259d6f54633.js

**Other relevant information:**
webpack version: 5.50.0
Node.js version: 12.16.2
Operating System: Linux
Additional tools:

**Comment by sokra** [comment]
We intentionally no longer name vender split chunks by the name of the origin chunk. vendor chunks might be shared by multiple chunks, adding a new chunk which shares an existing vendor chunk should not affect the filename/content of the vendor chunk, since this causes unnecessary invalidations.

You could restore the old behavior with a custom `splitChunks.cacheGroups.xxx.name` option, but I won't recommend doing that.
